Jenkinsfile

```diff
@@ -33,30 +33,34 @@ slack = new Slack()
 //
 // Smoke tests are CI-friendly test configuration.  Currently-supported Python version + modern C*/DSE instances.
 // We also avoid cython since it's tested as part of the nightlies.
+DEFAULT_CASSANDRA = ['2.1', '2.2', '3.0', '3.11', '4.0']
+DEFAULT_DSE = ['dse-5.0.15', 'dse-5.1.35', 'dse-6.0.18', 'dse-6.7.17', 'dse-6.8.30']
+DEFAULT_RUNTIME = ['2.7.18', '3.5.9', '3.6.10', '3.7.7', '3.8.3']
+DEFAULT_CYTHON = ["True", "False"]
 matrices = [
   "FULL": [
-    "SERVER": ['2.1', '2.2', '3.0', '3.11', '4.0', 'dse-5.0.15', 'dse-5.1.35', 'dse-6.0.18', 'dse-6.7.17', 'dse-6.8.30'],
-    "RUNTIME": ['2.7.18', '3.5.9', '3.6.10', '3.7.7', '3.8.3'],
-    "CYTHON": ["True", "False"]
+    "SERVER": DEFAULT_CASSANDRA + DEFAULT_DSE,
+    "RUNTIME": DEFAULT_RUNTIME,
+    "CYTHON": DEFAULT_CYTHON
   ],
   "DEVELOP": [
     "SERVER": ['2.1', '3.11', 'dse-6.8.30'],
     "RUNTIME": ['2.7.18', '3.6.10'],
-    "CYTHON": ["True", "False"]
+    "CYTHON": DEFAULT_CYTHON
   ],
   "CASSANDRA": [
-    "SERVER": ['2.1', '2.2', '3.0', '3.11', '4.0'],
-    "RUNTIME": ['2.7.18', '3.5.9', '3.6.10', '3.7.7', '3.8.3'],
-    "CYTHON": ["True", "False"]
+    "SERVER": DEFAULT_CASSANDRA,
+    "RUNTIME": DEFAULT_RUNTIME,
+    "CYTHON": DEFAULT_CYTHON
   ],
   "DSE": [
-    "SERVER": ['dse-5.0.15', 'dse-5.1.35', 'dse-6.0.18', 'dse-6.7.17', 'dse-6.8.30'],
-    "RUNTIME": ['2.7.18', '3.5.9', '3.6.10', '3.7.7', '3.8.3'],
-    "CYTHON": ["True", "False"]
+    "SERVER": DEFAULT_DSE,
+    "RUNTIME": DEFAULT_RUNTIME,
+    "CYTHON": DEFAULT_CYTHON
   ],
   "SMOKE": [
-    "SERVER": ['3.11', '4.0', 'dse-6.8.30'],
-    "RUNTIME": ['3.7.7', '3.8.3'],
+    "SERVER": DEFAULT_CASSANDRA.takeRight(2) + DEFAULT_DSE.takeRight(1),
+    "RUNTIME": DEFAULT_RUNTIME.takeRight(2),
     "CYTHON": ["False"]
```

Comment:
Just reducing some redundancy while we're here + making things a bit more declarative